src/catalog/product-configurations.md

```diff
@@ -9,3 +9,17 @@ _Configurations Section_
 
 ![]({% link images/images/product-configurations-hoodie.png %}){: .zoom}
 _Product Configurations_
+
+## Field reference
+
+|Field|Description|
+|--- |--- |
+|Image|Product image.|
```

Comment:
Generally, we don't use periods in these table descriptions, unless it is an actual sentence.